ext/mini_racer_extension/extconf.rb

```diff
@@ -1,3 +1,11 @@
+if defined?(TruffleRuby)
+  File.open("Makefile", "w") do |mf|
+      mf.puts "# Dummy makefile for TruffleRuby"
+      mf.puts "all install::\n"
+  end
+  return
+end
```

Comment:
Can you use the pattern like this one?
https://github.com/ruby/io-wait/pull/9/files#diff-b036344c1f726e5f63c8516ffb6412ee007186a02b167cbe1024e76c236204efR5
`File.write("Makefile", dummy_makefile($srcdir).join(""))`

Reply:
Fixed I updated both `extconf.rb` and also used the Rakefile pattern there as well.